src/components/tier.jsx

```diff
@@ -60,7 +61,7 @@ class Tier extends React.Component {
 
     return (
       <div>
-        <div className="tier ${this.state.tierId}">
+        <div className="tier">
           Tier {this.state.tierId}
           {itemsArray}
         </div>
```

Comment:
Not sure if you also meant to import ./tier.css to use on line 64, also your css class is capital T but classname on 64 is lowercase, otherwise looks great!

Reply:
Good catches, thank you!